source/manual/alerts/clamav-definitions-out-of-date.html.md

```diff
@@ -4,15 +4,16 @@ title: ClamAV definitions out of date
 parent: "/manual.html"
 layout: manual_layout
 section: Icinga alerts
-last_reviewed_on: 2019-10-01
+last_reviewed_on: 2020-07-16
 review_in: 6 months
 ---
 
 This could be because of a number of reasons. Check that the database is up to
-date by calling the freshclam command directly with verbose:
+date by [logging into a backend machine][] and calling the freshclam command directly with verbose:
```

Comment:
I think you could get away without linking this, as you've copied the relevant command into the bash snippet below.